src/main/java/eci/ieti/leafOut/backEnd/model/Location.java

```diff
@@ -1,35 +1,16 @@
-package eci.ieti.leafout.backend.model;
+package eci.ieti.leafout.backEnd.model;
 
-public class Location {
 
+import lombok.*;
 
+@Builder
```

Comment:
Lombok library is great for avoiding boilerplate code, but have in mind you may face issues when validating the coverage of your unit tests.